troubleshoot/elasticsearch/file-based-recovery.md

```diff
@@ -0,0 +1,217 @@
+---
+applies_to:
+  deployment:
+    eck: ga 
+    self: ga
+products:
+  - id: elasticsearch
+  - id: cloud-kubernetes
+---
+
+# File-based recovery
```

Comment:
can we clarify exactly what is being recovered here? I think it would be hard to self-discover this topic based on the parent page and this title. would it make sense to say "File-based access recovery"?